discord/guild.py

```diff
@@ -2009,6 +2044,140 @@ def convert(d):
 
         return [convert(d) for d in data]
 
+    async def fetch_stickers(self) -> List[GuildSticker]:
+        r"""|coro|
+
+        Retrieves a list of all :class:`Sticker`\s for the guild.
+
+        .. note::
+
+            This method is an API call. For general usage, consider :attr:`stickers` instead.
+
+        Raises
+        ---------
+        HTTPException
+            An error occurred fetching the stickers.
+
+        Returns
+        --------
+        List[:class:`Sticker`]
+            The retrieved stickers.
+        """
+        data = await self._state.http.get_all_guild_stickers(self.id)
+        return [GuildSticker(state=self._state, data=d) for d in data]
+
+    async def fetch_sticker(self, sticker_id: int, /) -> GuildSticker:
+        """|coro|
+
+        Retrieves a custom :class:`Sticker` from the guild.
+
+        .. note::
+
+            This method is an API call.
+            For general usage, consider iterating over :attr:`stickers` instead.
+
+        Parameters
+        -------------
+        sticker_id: :class:`int`
+            The sticker's ID.
+
+        Raises
+        ---------
+        NotFound
+            The sticker requested could not be found.
+        HTTPException
+            An error occurred fetching the sticker.
+
+        Returns
+        --------
+        :class:`Sticker`
+            The retrieved sticker.
+        """
+        data = await self._state.http.get_guild_sticker(self.id, sticker_id)
+        return GuildSticker(state=self._state, data=data)
+
+    async def create_sticker(
+        self,
+        *,
+        name: str,
+        description: str = MISSING,
+        emoji: str,
+        file: File,
+        reason: Optional[str] = None,
+    ) -> GuildSticker:
+        """|coro|
+
+        Creates a :class:`Sticker`s for the guild.
```

Comment:
```suggestion
        Creates a :class:`Sticker` for the guild.
```

Hehe. Maybe also want to specify "custom"?